shoot-client/path-controller.sh

```diff
@@ -67,7 +67,14 @@ function pingAllShootClients() {
     set +e
     for (( c=0; c<$HA_VPN_CLIENTS; c++ )); do
         ip="${bondPrefix}.$((bondStart+c+2))"
-        ping -W 2 -w 2 -c 1 $ip > /dev/null &
+        # Only output the second line if the time is at least 100 ms instead of the following:
+        # PING 192.168.123.194 (192.168.123.194): 56 data bytes
+        # 64 bytes from 192.168.123.194: seq=0 ttl=64 time=13.658 ms
+        #
+        # --- 192.168.123.194 ping statistics ---
+        # 1 packets transmitted, 1 packets received, 0% packet loss
+        # round-trip min/avg/max = 13.658/13.658/13.658 ms
+        ping -W 2 -w 2 -c 1 $ip | sed '/\(^PING.*\|^---.*---$\|.*1 packets received.*\|^round-trip.*\|^$\|.*time=.\?..... ms$\)/d' &
         ping_pid[$ip]=$!
```

Comment:
I'm no expert for bash scripts, but I have doubts that `$!` contains the the PID of the ping process.
On Mac I get the PID of the sed instead:
```bash
❯ ping  -c 1 127.0.0.1 | sed '/\(^PING.*\|^---.*---$\|.*1 packets received.*\|^round-trip.*\|^$\|.*time=.\?..... ms$\)/d' &
[1] 33210 33211
[1]  + 33210 done       ping -c 1 127.0.0.1 |
       33211 done       sed

❯ echo $!
33211
```

Reply:
Good catch. I reworked the solution based on your input. The approach using post-processing of the ping output works fine.

The following lines are retained:
- packet loss
- roundtrip delays >= 100 ms

```
2024-06-04 14:43:11 | {"log":"1 packets transmitted, 0 packets received, 100% packet loss"}
...
2024-06-04 14:43:04 | {"log":"64 bytes from 192.168.123.195: seq=0 ttl=64 time=158.881 ms"}
```

WDYT?